src/Propel/Generator/Platform/PgsqlPlatform.php

```diff
@@ -782,16 +788,18 @@ public function getUsingCast(Column $fromColumn, Column $toColumn): string
             return " USING decode(CAST($name as text), 'escape')";
         }
 
-        if ($fromSqlType === 'DATE' && $toSqlType === 'TIME') {
-            return ' USING NULL';
-        }
```

Comment:
This DATE and TIME handling is dropped, why?

Reply:
The function returns ` USING NULL` per default, so this was just an extra step to get to the same result.